.gitignore

```diff
@@ -6,3 +6,5 @@ dist
 *.umd.js
 yarn-error.log
 .env
+
+.idea/
```

Comment:
Make a separate PR for this

Reply:
it probably should just be reverted actually, local files should be in a global gitignore